docs/OncoTree-Version-Converter.md

```diff
@@ -0,0 +1,17 @@
+# OncoTree Version Converter Tool
+
+The OncoTree Version Converter tool was developed to facilitate forwards- and backwards-mapping between OncoTree release versions. 
+
+## Running the tool
+
+The OncoTree Version Converter tool can be run with the following command:
+
+```
+python ./scripts/convert_oncotree_version.py --input-file <path/to/input/file> --output-file <path/to/output/file> --source-version <source_oncotree_version> --target-version <target_oncotree_version>
```

Comment:
this depends on the user's current working directory. for inexperienced unix users maybe it would be best to not give any path to the script, and they will run the tool in the same directory as the tool?  Or we could use the same notation as the other arguments ... python <path/to/convert_conotree_version.py> --input-file <....> ...